staff/fede-gomez/test-sass/style.css

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Is empty.

Look at the files you have uploaded, this is under test-sass directory not under cv-styled directory

Reply:
You are right. It is empty. I haven't worked on this. It's a WIP :) Thanks though!